### DISCUSSION TITLE
I just paid for a single licence, how long does the license last?

### DISCUSSION THREAD

**Discussion by alecos71** [fancyapps/ui#465] [General]
I just paid for a single license, how long does the license last? I paid € 35.38, single license one time payment.

I don't have a commercial website, my website is personal even though I develop free software offered for free.

Kind regards.

Alessandro

**Comment by fancyapps** [comment]
Hi,

This is a perpetual license, e.g. it's a one-time fee and you can use it indefinitely.

Thanks for choosing Fancyapps,
Janis

**Reply by alecos71** [reply]
Thanks!

Have a nice day.

Kind regards.

Alessandro

**Comment by alecos71** [comment]


**Reply by fancyapps** [reply]
Hi,

First, you could have contacted me directly using info@fancyapps.com email address. On the account page, you can only view your bills, nothing more. 

1) If xxx@gmail.com is your email address, then I see no reason why you can't log in. Everything seems to be working fine (this project uses Firebase Authentication).
2) Everything related to billing and payments is handled by Paddle. If you have any specific questions or requests, please contact them directly.

Janis

**Reply by alecos71** [reply]
Ok, yes, it's my email address, thanks very much for the help! Can you, now, hide my email address in your comment? I don't want spammers into my mailbox, thanks.

For future help, I'll contact you directly, sorry for the mistake.

Kind regards.

Alessandro

**Reply by alecos71** [reply]
I wrote you directly, let's me know if you received my email.

Thank you!

Kind regards.

Alessandro

**Comment by alecos71** [comment]
Problem solved.

Happy evening.

Thanks Janis.
